### PR TITLE
Add gitlab ci and document setup process

### LINKED ISSUE #15: Remove docker requirement
I tried deploying this but it requires docker for some reason. Is that a necessary requirement?

### PR DESCRIPTION
**Pull request by carlin-q-scott**
Closes #15

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+image: nikolaik/python-nodejs:python3.8-nodejs12
```
Comment (aexvir) [inline .gitlab-ci.yml]:
```suggestion
image: kiwicom/sls
```

I'd rather use our own image for this :)
It doesn't have python 3.8 though, but we don't need it at the moment

Comment (aexvir) [reply .gitlab-ci.yml]: Here is the dockerfile for that image: https://github.com/kiwicom/dockerfiles/blob/master/sls/Dockerfile
This is actually the image we use internally for sls deployments

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+deploy:
+  stage: deploy
+  environment: 
+    name: development
+    on_stop: stop_environment
+  before_script: &setup_serverless
```
Comment (aexvir) [inline .gitlab-ci.yml]: All this can be avoided then

Comment (carlin-q-scott) [reply .gitlab-ci.yml]: I removed the before_script

**Comment by carlin-q-scott** [comment]
I reverted back to python 3.6 so that it's compatible with the docker image you recommended.

**Pull request merged**